tests/samplers/test_rejection_sampler.py

```diff
@@ -198,6 +200,59 @@ def test_deterministic_when_seeded(k: int, vocab_size: int, batch_size: int,
                 assert torch.equal(results[j][i], results[0][i])
 
 
+@pytest.mark.parametrize("k", [1, 3, 6])
+@pytest.mark.parametrize("vocab_size", [30_000, 50_000])
+@pytest.mark.parametrize("batch_size", [1, 8, 32, 128])
+@pytest.mark.parametrize("device", CUDA_DEVICES)
+@torch.inference_mode()
+def test_nonflashinfer_backend(k: int, vocab_size: int, batch_size: int,
```

Comment:
Can you add a docstring explaining the test methodology? e.g. as a maintainer, I want to know what value this test provides, how it tests what it tests